frontend/src/app/history/history.component.html

```diff
@@ -0,0 +1,83 @@
+                        </p-inputIcon>
+                        <input
+                            pInputText
+                            type="text"
+                            (input)="dt.filterGlobal($any($event.target).value, 'contains')"
+                            placeholder="Search keyword" />
+                    </p-iconField>
+                </div>
+            </ng-template>
+            <ng-template pTemplate="header" let-columns>
+                <tr>
+                    <th pSortableColumn="question" style="width: 20%">
+                        Question<p-sortIcon field="question"></p-sortIcon>
+                    </th>
+                    <th pSortableColumn="difficulty" style="width: 15%">
+                        Difficulty<p-sortIcon field="difficulty"></p-sortIcon>
+                    </th>
+                    <th pSortableColumn="topics" style="width: 24%">Topics<p-sortIcon field="topics"></p-sortIcon></th>
+                    <th pSortableColumn="collaborator" style="width: 17%">
+                        Collaborator<p-sortIcon field="collaborator"></p-sortIcon>
+                    </th>
+                    <th pSortableColumn="status" style="width: 12%">Status<p-sortIcon field="status"></p-sortIcon></th>
+                    <th pSortableColumn="time" style="width: 12%">Time<p-sortIcon field="time"></p-sortIcon></th>
+                </tr>
+            </ng-template>
+            <ng-template pTemplate="body" let-history>
+                <tr (click)="onRowSelect(history)">
+                    <td>{{ history.question.title }}</td>
+                    <td>{{ history.difficulty }}</td>
+                    <td>{{ history.topics.join(', ') }}</td>
+                    <td>{{ history.collaborator }}</td>
+                    <td>
+                        @if (history.status === 'COMPLETED') {
+                            <i class="pi pi-check" style="color: green; font-size: large"></i>
+                        } @else if (history.status === 'FORFEITED') {
+                            <i class="pi pi-times" style="color: red; font-size: large"></i>
+                        } @else if (history.status === 'IN_PROGRESS') {
+                            <i class="pi pi-spin pi-spinner" style="color: white; font-size: large"></i>
+                        }
+                    </td>
+                    <td>{{ history.time }}</td>
+                </tr>
+            </ng-template>
+        </p-table>
+    </div>
+    <div class="sliding-panel" [class.open]="isPanelVisible">
+        <div class="panel-header">
+            <h3>{{ panelHistory?.question?.title }}</h3>
+            <p-button
+                icon="pi pi-times"
+                severity="secondary"
+                label="Close"
+                (onClick)="closePanel()"
+                class="p-button-text" />
+        </div>
+        <div class="panel-content">
+            <p class="white-space: pre-wrap">{{ panelHistory?.question?.description }}</p>
```

Comment:
```suggestion
            <p style="white-space: pre-wrap">{{ panelHistory?.question?.description }}</p>
```

Think the styling here is wrong